### DISCUSSION TITLE
Conflicting settings question

### DISCUSSION THREAD

**Discussion by Spit29** [gopro/labs#228] [Q&A]
Hi there, I was considering installing the labs firmware on my hero 11 mini. I was just wondering what happens if setup conflicting settings like hyperview and an 8:7 lens? Or 5.3k with 240fps? It seems all settings are available when that wouldn't work.. 

Also, does maximum shutter angle apply to all the presets? Or does it have to be scanned for each one? https://gopro.github.io/labs/control/maxshut/

Thanks!

**Comment by dnewman-gpsw** [comment]
Conflicting settings are ignored.  You can ask for 5.3K 240fps, the camera will still in it current mode. 

All the metadata hacks are global (oMwxyz=1), so max shutter angle effect all video modes.

**Reply by Spit29** [reply]
> Conflicting settings are ignored. You can ask for 5.3K 240fps, the camera will still in it current mode.
> 
> All the metadata hacks are global (oMwxyz=1), so max shutter angle effect all video modes.

Thank you!! Just 2 more questions if you could:
1. I have a hero 10 bones as well. Is there a way to just have video mode permanently? (remove picture and time lapse)? It's hard to tell what mode i'm on.. Will the qr code changes immediately show up on the gopro app once scanned?
2. Also with my drones should I be setting them to NOT go to sleep when using IMU detection? I'm guessing it only works when powered on. https://gopro.github.io/labs/control/imutrigger/

**Reply by dnewman-gpsw** [reply]
1. QR Code changes are immediate, add mV to the beginning of any QR Code will switch the camera the mode Video.  You can also use a boot command that ensure the camera in video mode.  Removing photo and timelapse is not an option.
2. Auto off on Bones is a little silly.  Scan command  oC to disable auto off.

**Reply by Spit29** [reply]
Would archive mode lock the hero bones into video mode? If I can just set it to 4k/60 and leave it there that would be perfect.
https://gopro.github.io/labs/control/archive/

**Reply by dnewman-gpsw** [reply]
I did try that and it worked, after I disabled the auto boot (which I always use).  It seems those two hacks share too much.  So it you use archive mode, it does work on Bones, if you aren't using WAKE=1  (default is !MWAKE=0).

**Reply by Spit29** [reply]
> I did try that and it worked, after I disabled the auto boot (which I always use). It seems those two hacks share too much. So it you use archive mode, it does work on Bones, if you aren't using WAKE=1 (default is !MWAKE=0).

Oh Great! So when the drone is powered on the bones will start in the OFF position !MWAKE=0. Pressing either button will start a video recording (In Archive mode). Holding the menu button will still turn it on correct? (So I can change video settings with other qr codes)

**Reply by dnewman-gpsw** [reply]
No to the last bit.  Archive mode is design so that no user action can change modes, so there is no long press for menu access.  You would have to disable archive to change settings, this is another reason to use a boot command instead.
